astro/src/content/blog/fusionauth-and-terraform.mdx

```diff
@@ -0,0 +1,64 @@
+---
+publish_date: 2025-05-15
+title: "Why Use Terraform With FusionAuth"
+description: 'Learn why using Terraform with FusionAuth is beneficial.'
+authors: Mark Robustelli
+image: /img/blogs/fusionauth-and-terraform/fusionauth-and-terraform.png
+categories: Tutorial
+tags: deployment, terraform, fusionauth, deployment
+excerpt_separator: "{/* more */}"
+---
+
+import Aside from '/src/components/Aside.astro';
+
+Let's be honest — the only thing better than a fully working auth system is never having to manually configure it again. Enter FusionAuth and Terraform.
+
+In this post, you will learn why you would want to use the Terraform provider for FusionAuth. You can ditch the manual clicks and start defining your identity resources as code.
```

Comment:
```suggestion
In this post, you will learn why you would want to use the Terraform provider for FusionAuth. I'm writing this today. because we recently released[ version 1.0.0 of the Terraform provider](https://github.com/fusionauth/terraform-provider-fusionauth). With Terraform, ditch the manual clicks and start defining your identity resources as code.
```